action.yaml

```diff
@@ -37,15 +41,16 @@ branding:
 runs:
   using: composite
   steps:
-    - name: Run minimum dependency check
+    - name: Run Dependency Test
       run: |
+        echo "Dependency Test strategy: ${{ inputs.strategy }}"
         if [ "${{ inputs.additional-env-vars }}" != "" ]
         then {
           echo -e "${{ inputs.additional-env-vars }}" > /tmp/dotenv.env
           export $(tr '\n' ' ' < /tmp/dotenv.env)
         }
         fi
-        Rscript ${GITHUB_ACTION_PATH}/script.R ${{ inputs.repository-path }} ${{ inputs.build-args }} ${{ inputs.check-args }}
+        Rscript ${GITHUB_ACTION_PATH}/script.R ${{ inputs.repository-path }} ${{ inputs.build-args }} ${{ inputs.check-args }} ${{ inputs.strategy }}
```

Comment:
@pawelru I would need your help with using this argument in the script.

Reply:
Yes I will help you on this.

Reply:
done